hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashPolicyOzone.java

```diff
@@ -118,6 +120,25 @@ public Runnable getEmptier() throws IOException {
         emptierInterval);
   }
 
+  @Override
+  public boolean moveToTrash(Path path) throws IOException {
+    this.fs.getFileStatus(path);
+    Path trashRoot = this.fs.getTrashRoot(path);
+    Path trashCurrent = new Path(trashRoot, CURRENT);
+
+    String key = path.toUri().getPath();
+    String trashRootKey = trashRoot.toUri().getPath();
+
```

Comment:
Can we please add a debug level log that prints values for` key` and `trashRootKey.`